package.json

```diff
@@ -12,9 +12,9 @@
     "prettier": "npx prettier . --write"
   },
   "dependencies": {
-    "@abgov/react-components": "6.9.0-alpha.2",
-    "@abgov/ui-components-common": "1.9.0-alpha.1",
-    "@abgov/web-components": "1.39.0-alpha.2",
+    "@abgov/react-components": "6.9.3-dev.2",
+    "@abgov/ui-components-common": "1.9.1-dev.1",
+    "@abgov/web-components": "1.39.3-dev.2",
```

Comment:
These should be set to the `next` version.

React: 6.9.3-next.1
Angular: 4.9.1-next.1
Web: 1.39.3-next.1